qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -321,6 +320,8 @@
             diskinfo_guest_cmd = 'lsblk -Jp -o KNAME,PKNAME,TYPE | grep %s|awk "NR==1"'
             rawdisk_guest_cmd = 'lsblk -Jp -o KNAME,TYPE | grep %s|awk "NR==1"'
             cmd_get_disk_alias = 'lsblk -o NAME %s |grep -v NAME'
+            Windows:
+                cmd_get_diskinfo = wmic diskdrive where "DeviceID='%s'" get Name,Partitions | findstr /v /i Name
```

Comment:
`Partitions` Seems useless in your check, why not only get `Name`?

Reply:
before I wanted to check partitions at the beginning, but deleted it now. code updated.